ydb/mvp/core/mvp_tokens.h

```diff
@@ -81,7 +71,7 @@ class TMvpTokenator : public NActors::TActorBootstrapped<TMvpTokenator> {
         using TEvUpdateStaticCredentialsToken = TEvUpdateToken<EvUpdateStaticCredentialsToken, Ydb::Auth::LoginResponse>;
     };
 
-    TMvpTokenator(NMvp::TTokensConfig tokensConfig, const NActors::TActorId& httpProxy, const EAuthProfile authProfile);
+    TMvpTokenator(NMvp::TTokensConfig tokensConfig, const NActors::TActorId& httpProxy, const EAccessServiceType accessServiceType);
```

Comment:
Parameter `const EAccessServiceType accessServiceType` is equivalent here to `EAccessServiceType accessServiceType`, because it is passed by value. I don't suggest to change it, but just for information if you wanted to make some different things :)